### PR TITLE
adding converted release notes and updating conf.py

### PR DESCRIPTION
**Pull request by stanzikratel**
updated conf.py
updated index.rst
added release note folder and converted release notes

### CODE REVIEW

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

**Comment by stretchservers** [comment]
Can one of the admins verify this patch?

File: api-docs/rst/dev-guide/glossary.rst [hunk 1]
```diff
@@ -1,3 +1,5 @@
+.. _glossary:
+
 Glossary 
 ----------
```
Comment (meker12) [inline api-docs/rst/dev-guide/glossary.rst]: We removed the glossaries from all the dev guides and rolled the terms into the common glossary.  Can you remove this one?

File: api-docs/rst/dev-guide/release-notes.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Learn about new features, enhancements, known issues, resolved issues, and
+other important details about |apiservice| |version| service updates.
+
+.. note:: For information about using the API, see the
+   :ref:`documentation overview <index>`.
+
+.. toctree::
+   :maxdepth: 2
+
+   November 20, 2013 <release-notes/as_20131120.rst>
```
Comment (meker12) [inline api-docs/rst/dev-guide/release-notes.rst]: In the index.rst files, you don't have to include the *.rst extension in the file list.  If the build doesn't give any errors, then I think it isn't a problem, but can you verify the build.

**Comment by meker12** [comment]
Closing this request because content has been included in PR #1805 .

**Pull request closed without merging**